resource/service_freebsd.go

```diff
@@ -0,0 +1,118 @@
+// +build freebsd
+
+package resource
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+// Service type is a resource which manages services on a
+// FreeBSD.
+//
+// Example:
+//   svc = resource.service.new("nginx")
+//   svc.state = "running"
+//   svc.enable = true
+//   svc.rcvar = "nginx_enable"
+type Service struct {
+	Base
+
+	// If true then enable the service during boot-time
+	Enable bool `luar:"enable"`
+
+	// RCVar (see rc.subr(8)), set to {svcname}_enable by default.
+	// If service doesn't define rcvar, you should set svc.rcvar = "".
+	RCVar string `luar:"rcvar"`
+}
+
+// NewService creates a new resource for managing services
+// using systemd on a GNU/Linux system
```

Comment:
Can you please update the comment that this resource is actually for FreeBSD systems?